.github/configuration/upgrade-test-configuration.yml

```diff
@@ -38,6 +38,16 @@ upgrade-version: [[
     upgrade-type: minor
   }
 ],
+[
+  {
+    version: 14.8,
+    upgrade-type: null
+  },
+  {
+    version: target.latest,
+    upgrade-type: minor
+  }
+],
```

Comment:
Let's not add a new workflow, instead update the above entry (14.6 -> latest) to 14.8 -> latest.